pkg/flowaggregator/flowaggregator.go

```diff
@@ -281,7 +278,34 @@ func (fag *flowAggregator) Run(stopCh <-chan struct{}) {
 					continue
 				}
 			}
-			err := fag.aggregationProcess.ForAllRecordsDo(fag.sendFlowKeyRecords)
+			dataSet := ipfix.NewSet(ipfixentities.Data, fag.templateID, false)
+			sendFlowKeyRecord := func(key ipfixintermediate.FlowKey, record ipfixintermediate.AggregationFlowRecord) error {
+				if !record.ReadyToSend {
+					klog.V(4).Info("Skip sending record that is not correlated.")
+					return nil
+				}
+				elements := make([]*ipfixentities.InfoElementWithValue, 0)
+				for _, ieString := range fag.infoElementsOrder {
+					element, exist := record.Record.GetInfoElementWithValue(ieString)
+					if !exist {
+						return fmt.Errorf("%s not present.", ieString)
+					}
+					elements = append(elements, element)
+				}
+				err := dataSet.AddRecord(elements, fag.templateID)
+				if err != nil {
+					return fmt.Errorf("AddRecord failed, error: %v", err)
+				}
+
+				bytesSent, err := fag.exportingProcess.SendSet(dataSet.GetSet())
+				if err != nil {
+					return fmt.Errorf("Send data record for key %+v failed, error: %v", key, err)
+				}
+				fag.sentFlowKeys = append(fag.sentFlowKeys, &key)
+				klog.V(4).Infof("Sent key: %+v, sent bytes: %d\n", key, bytesSent)
+				return nil
```

Comment:
yes. Maybe it is useful to expose the pathMTU from the exporter. What do you think?

Reply:
Yes it will be useful to do that in next release. However, we can do the same thing as pkg/agent/flowexporter/exporter/exporter.go. Can we do the same thing with that workaround?